test/module.test.js

```diff
@@ -18,4 +18,4 @@ it('adds e to the user PATH', () => {
 it('can execute e', () => {
   const info = execSync('e --help');
   assert.match(info.toString(), /Electron build tool/);
-}).timeout(10000);
+}).timeout(30000);
```

Comment:
This seems to be significantly slower on Windows on GHA, specifically with Node.js 18.18 - will dig into it in a follow-up.